test/nginx_system_test.sh

```diff
@@ -1263,6 +1277,10 @@ OUT=$(cat "$ERROR_LOG" \
     | grep -v "\\[error\\].*Failed to make directory*" \
     | grep -v "\\[error\\].*Could not create directories*" \
     | grep -v "\\[error\\].*opening temp file: No such file or directory.*" \
+    | grep -v "\\[error\\].*unexpected response.*" \
```

Comment:
@crowell was (re-)adding this line really needed? I was hoping that https://github.com/pagespeed/ngx_pagespeed/pull/922 would have fixed this -- have you been testing with that change included?